debugfs/dump.c

```diff
@@ -338,7 +351,6 @@ static int rdump_dirent(struct ext2_dir_entry *dirent,
 {
 	char name[EXT2_NAME_LEN + 1];
 	int thislen;
-	const char *dumproot = private;
```

Comment:
It would be more readable to cast `private` here up front:
`rdump_dirent_private *dirent_private = (rdump_dirent_private *) private;`

And then use `dirent_private` in the `rdump_inode()` call below.